modules/kontraktor-reallive/src/main/java/org/nustaq/reallive/api/SafeRealLiveTable.java

```diff
@@ -12,7 +12,7 @@
  */
 public interface SafeRealLiveTable extends ChangeReceiver, SafeChangeStream, SafeRealLiveStreamActor {
 
-    IPromise<?> ping();
+    IPromise<Object> ping();
```

Comment:
if capture ? is returned, following does not work anymore:
pingPromise.then(anotherPromise)
as it cannot find Callback<T> with Callback<?>

quickfix: pingPromise.then(() -> anotherPromise.resolve())